mintpy/utils/plot.py

```diff
@@ -917,7 +917,7 @@ def plot_coherence_matrix(ax, date12List, cohList, date12List_drop=[], p_dict={}
     return ax, coh_mat, im
 
 
-def read_dem(dem_file, pix_box=None, geo_box=None, print_msg=True):
+def read_dem(dem_file, pix_box=None, geo_box=None, print_msg=True, multilook_num=1):
```

Comment:
`multilook_num` is not used here. And note that the DEM here could have a higher resolution than the data, so it's fine to not multilook.

Reply:
this func is used only for single plot, which does not use multilook_num. Revert this change.